ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -421,6 +421,20 @@ def test_ipa_healthcheck_remove(self):
         tasks.uninstall_packages(self.master, HEALTHCHECK_PKG)
         tasks.uninstall_packages(self.replicas[0], HEALTHCHECK_PKG)
 
+    def test_ipa_healthcheck_output_indent(self):
+        """
+        This test case checks whether default (2) indentation is applied
+        to output without it being implicitly stated
+        """
+        output = self.master.run_command(["ipa-healthcheck"], raiseonerr=False)
+        output_list_of_jsons = output.stdout_text.split(",")
+        if not len(output_list_of_jsons):  # there is only one json in list
+            output_list_of_jsons = [output.stdout_text.strip("[]")]
+        for output_json in output_list_of_jsons:
+            if output_json:
+                assert str(json.dumps(output_json.strip(),
+                                      indent=2)) == output_json.strip()
```

Comment:
The output of healthcheck is a single JSON object. There is no need to each entry separately.

It is not possible for ipa-healthcheck to return an empty list in the default case.

This should use the newish run_healthcheck method. I suggested previously to add a raw option so the unprocessed JSON can be returned.

Reply:
@rcritten so, if I understand correctly, as the output is begins with "[" and ends with "]", I should add "{" and "}" and read it as json?